src/Tizen.NUI.Scene3D/src/public/Controls/Panel.cs

```diff
@@ -0,0 +1,377 @@
+                pValue.Dispose();
+                return transparent;
+            }
+            set
+            {
+                var transparent = new Tizen.NUI.PropertyValue(value);
+                SetProperty(Interop.Panel.PropertyTransparentIndexGet(), transparent);
+                transparent.Dispose();
+            }
+        }
+
+        /// <summary>
+        /// Whether the content is rendered as double sided or not.
+        /// Default value is false.
+        /// </summary>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool DoubleSided
+        {
+            get
+            {
+                bool doubleSided = false;
+                Tizen.NUI.PropertyValue pValue = GetProperty(Interop.Panel.PropertyDoubleSidedIndexGet());
+                pValue.Get(out doubleSided);
+                pValue.Dispose();
+                return doubleSided;
+            }
+            set
+            {
+                var doubleSided = new Tizen.NUI.PropertyValue(value);
+                SetProperty(Interop.Panel.PropertyDoubleSidedIndexGet(), doubleSided);
+                doubleSided.Dispose();
+            }
+        }
+
+        /// <summary>
+        /// Whether to use back face plane or not.
+        /// If this property is true, an opaque plane will be displayed when viewed from behind the Panel.
+        /// Default value is true.
+        /// Default back face plane color is white.
+        /// </summary>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool UsingBackFacePlane
+        {
+            get
+            {
+                bool isUsingBackFacePlane = true;
+                Tizen.NUI.PropertyValue pValue = GetProperty(Interop.Panel.PropertyUseBackFacePlaneIndexGet());
+                pValue.Get(out isUsingBackFacePlane);
+                pValue.Dispose();
+                return isUsingBackFacePlane;
+            }
+            set
+            {
+                var isUsingBackFacePlane = new Tizen.NUI.PropertyValue(value);
+                SetProperty(Interop.Panel.PropertyUseBackFacePlaneIndexGet(), isUsingBackFacePlane);
+                isUsingBackFacePlane.Dispose();
+            }
+        }
+
+        /// <summary>
+        /// Color of back face plane.
+        /// Default color is white.
+        /// </summary>
+        /// <remarks>
+        /// Because back face plane is always opaque, alpha channel is ignored.
+        /// </remarks>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Color BackFacePlaneColor
+        {
+            get
+            {
+                Vector3 backFacePlaneColorVector3 = new Vector3(0.0f, 0.0f, 0.0f);
```

Comment:
```suggestion
                Vector3 backFacePlaneColorVector3 = new Vector3(1.0f, 1.0f, 1.0f);
```

Please make ensure that we return White for accident cases (like pValue is null)

Reply:
! Very Thank you.